.github/workflows/create-release.yaml

```diff
@@ -12,6 +12,10 @@ on:
         type: boolean
         default: false
 
+permissions: # used by build images steps
+  id-token: write # This is required for requesting the JWT token
+  contents: read # This is required for actions/checkout
```

Comment:
It needs to be `contents: write` without it the workflow could not push anything to the repo, e.g. tag or draft release.